spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/AbstractKafkaBinderTests.java

```diff
@@ -541,8 +590,8 @@ public void testDefaultConsumerStartsAtEarliest() throws Exception {
 		Binder binder = getBinder(createConfigurationProperties());
 		GenericApplicationContext context = new GenericApplicationContext();
 		context.refresh();
-		//binder.setApplicationContext(context);
-		//binder.afterPropertiesSet();
+		// binder.setApplicationContext(context);
+		// binder.afterPropertiesSet();
```

Comment:
Several code lines up to the `context` are out of use.
Good candidates for removal?